Dockerfile

```diff
@@ -25,6 +25,8 @@ RUN playwright install-deps
 COPY ./data ./data
 COPY ./src ./src
 COPY ./cli ./cli
+COPY ./.git ./.git
+COPY ./.pre-commit-config.yaml ./.flake8 ./.gitignore ./
```

Comment:
Adding test related code to the docker image is something we need to fix in general across repos. Will raise a tech debt issue.